website/docs/r/emr_cluster.html.markdown

```diff
@@ -148,6 +146,98 @@ Started](https://docs.aws.amazon.com/ElasticMapReduce/latest/ManagementGuide/emr
 guide for more information on these IAM roles. There is also a fully-bootable
 example Terraform configuration at the bottom of this page.
 
+## Instance Fleet
+
+```hcl
+resource "aws_emr_cluster" "example" {
+  # ... other configuration ...
+  master_instance_fleet {
+    instance_fleet_type = "MASTER"
```

Comment:
when I tried to use this config, it failed with
```
Error: Unsupported argument

  on emr.tf line 23, in resource "aws_emr_cluster" "cluster":
  23:     instance_fleet_type = "MASTER"
```
after I removed "instance_fleet_type" it worked. Most likely you need to update this doc.
same for core and task

Reply:
yes, I just removed the `instance_fleet_type` for the instance_fleet and forget to update the document.